tests/integration/bulk_import/conftest.py

```diff
@@ -0,0 +1,276 @@
+import uuid
+import pytest
+
+from labelbox.schema.labeling_frontend import LabelingFrontend
+
+IMG_URL = "https://picsum.photos/200/300"
+
+
+@pytest.fixture
+def ontology():
+    bbox_tool = {
+        'required':
+            False,
+        'name':
+            'bbox',
+        'tool':
+            'rectangle',
+        'color':
+            '#a23030',
+        'classifications': [{
+            'required': False,
+            'instructions': 'nested',
+            'name': 'nested',
+            'type': 'radio',
+            'options': [{
+                'label': 'radio_option_1',
+                'value': 'radio_value_1'
+            }]
+        }]
+    }
+    polygon_tool = {
+        'required': False,
+        'name': 'polygon',
+        'tool': 'polygon',
+        'color': '#FF34FF',
+        'classifications': []
+    }
+    polyline_tool = {
+        'required': False,
+        'name': 'polyline',
+        'tool': 'line',
+        'color': '#FF4A46',
+        'classifications': []
+    }
+    point_tool = {
+        'required': False,
+        'name': 'point--',
+        'tool': 'point',
+        'color': '#008941',
+        'classifications': []
+    }
+    entity_tool = {
+        'required': False,
+        'name': 'entity--',
+        'tool': 'named-entity',
+        'color': '#006FA6',
+        'classifications': []
+    }
+    segmentation_tool = {
+        'required': False,
+        'name': 'segmentation--',
+        'tool': 'superpixel',
+        'color': '#A30059',
+        'classifications': []
+    }
+    checklist = {
+        'required':
+            False,
+        'instructions':
+            'checklist',
+        'name':
+            'checklist',
+        'type':
+            'checklist',
+        'options': [{
+            'label': 'option1',
+            'value': 'option1'
+        }, {
+            'label': 'option2',
+            'value': 'option2'
+        }, {
+            'label': 'optionN',
+            'value': 'optionn'
+        }]
```

Comment:
I'd say this formatting is so-so... Values can be inlined here most of the time, resulting in twice as less rows and improved readability. I guess it's a result of automatic formatting, and I'm against it.

Reply:
Yes, I hope we can. Personally I hate this kind of unreadable mess. But, like I mentioned, I forgot it was mandatory for the build to pass. Let's consider outside of this PR.